tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -4,6 +4,8 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using System.Text;
+using System.Diagnostics;
+using System.Text.Json;
```

Comment:
I'm sure System.Text.Json is not longer used in the test. @tj-devel709  I have approved the PR but please remove it.